models/client.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+type Message struct {
```

Comment:
`Message` and `Options` are too general here, it's in `models` package so whoever see a struct `Mesage` might think about chat messages, push messages, or so. Use more specific name for AI related things